src/row.js

```diff
@@ -120,4 +149,35 @@ export class Row {
       this.isSeasonFour()
     );
   }
+
+  matchSummary = () => {
+    const outcome = this.won() ? 'Victory' : 'Defeat';
+    const mmr = this.mmr ? ` at ${this.mmr} MMR` : '';
+    const enemies = this.enemies();
+    return `${outcome} as ${this.allies()} vs ${enemies}${mmr}`;
+    // could also have shown isRanked/diffRating, day (endTime), zoneId...
+  };
+
+  allies = () =>
+    [
+      this.teamPlayerName1,
+      this.teamPlayerName2,
+      this.teamPlayerName3,
+      this.teamPlayerName4,
+      this.teamPlayerName5,
+    ]
+      .filter(a => !!a)
+      .join('/');
+
+  enemies = () => {
+    const enemies = [
+      enemy(this.enemyPlayerName1, this.enemyPlayerClass1),
+      enemy(this.enemyPlayerName2, this.enemyPlayerClass2),
+      enemy(this.enemyPlayerName3, this.enemyPlayerClass3),
+      enemy(this.enemyPlayerName4, this.enemyPlayerClass4),
+      enemy(this.enemyPlayerName5, this.enemyPlayerClass5),
+    ];
+
+    return enemies.filter(e => !!e).join(' and ');
```

Comment:
You applied the same logic for allies just above without creating a variable, please use the same code logic (either create a variable everywhere, or you don't)